docs/docs/dcap.md

```diff
@@ -1,96 +1,70 @@
 
-Conclave's bundled Azure client plugin will *only* be used if no other plugin has been found on the system.
-The runtime will use the first `.so` it encounters in the search order below:
-```
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
-/usr/lib/libdcap_quoteprov.so.1
-/usr/lib/libdcap_quoteprov.so
-```
-Should you decide to use a bundled version (recommended), ensure the files listed above don't exist (delete or rename
-them if necessary).
+`sudo apt-get install libsgx-dcap-default-qpl`
 
-!!!Important
+To use Intel's DCAP plugin, you need a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification). 
+You must also set up a Provisioning Certificate Caching Service (PCCS). Intel provides an example and some 
+instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). 
+If you like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs),
+then you might need to provide data in the correct format for Conclave. Please [let us know](mailto:conclave@r3.com) if 
+you need to set this up.
 
-    The bundled DCAP Azure client plugin works with all current generation Azure confidential VMs, as well as existing
-    CPUs. However, if Azure extends their offerings to include new platforms, or if you use a new generation system, 
-    then the bundled Azure client plugin may throw an error when attempting to load your enclave. In this case,
-    [follow the instructions below](#azure-client-plugin) to build/install an updated version of the Azure client 
-    plugin that supports the platform.
+### Azure DCAP Client
 
-You may want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
-```sh
-export AZDCAP_DEBUG_LOG_LEVEL=FATAL
-```
+Microsoft provides an Azure DCAP client, which you can use on Azure virtual machines. Azure's DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) 
+does not require a subscription. 
 
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
+You can use the following command to install Azure's DCAP client package:
 
-#### Azure client plugin
+`sudo apt-get install az-dcap-client`
 
-* Identify the currently installed DCAP client plugin. It will always have one of the following names: `libdcap_quoteprov.so.1` or `libdcap_quoteprov.so`. You might find other similarly named files, but they won't be used as a plugin.
-```sh
-ls /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so*
-ls /usr/lib/libdcap_quoteprov.so*
-```
+### Conclave-Azure Bundled Client
+
+Conclave provides a bundled Azure DCAP plugin, which is the recommended option for users. This plugin will be used 
```

Comment:
This is the recommended option for users using Azure virtual machine, can we make it explicit?